src/constraint_functions.jl

```diff
@@ -52,8 +52,11 @@ end
 """
     constraints_capacity_installed(m, n, 𝒯::TimeStructure, modeltype::EnergyModel)
 
-In general, it is prefered to have the capacity as a function of a variable given with a
-value of 1 in the field `n.Cap`.
+Function for limiting the the installed capacity to the available capacity.
```

Comment:
```suggestion
Constrain the installed capacity to the available capacity.
```
I suggest to avoid superfluous wording like "This function.../Function for..." when documenting functions.